src/jit/target.h

```diff
@@ -337,7 +337,13 @@ typedef unsigned short          regPairNoSmall; // arm: need 12 bits
 #if defined(_TARGET_X86_)
 
   #define CPU_LOAD_STORE_ARCH      0
+
+#ifdef LEGACY_BACKEND
   #define CPU_LONG_USES_REGPAIR    1
+#else
+  #define CPU_LONG_USES_REGPAIR    0
```

Comment:
Please add a comment that RyuJIt x86 doesn't allocate reg-pairs for long type vars and instead decomposes them into two int type vars during lowering.